addons/website_sale/static/src/scss/product_configurator.scss

```diff
@@ -250,6 +250,12 @@ label.css_attribute_color.css_not_available {
     padding: 0.075rem 0.75rem;
 }
 
+@include media-breakpoint-down(md) {
+    .td-img {
```

Comment:
I would add `.oe_advanced_configurator_modal` to be a little more specific, eg:

```py
    .oe_advanced_configurator_modal .td-img {
```
the other CSS for td-img is constrained to children of `.js_product` elements, so it seems odd to add a generic style for the class.

Reply:
Even after hours of searching I still have no clue why but despite the breakpoint being active, only the table body turned invisible, not the header (which had the exact same classes). There is probably something in there that makes the style too specific but I preferred adding style rather than removing constraints.

You're right that being more specific would be preferable though, I'll try .oe_advanced_configurator_modal.

Reply:
the table body rows have the class `js_product` and the header doesn't have it, and the CSS selector that hides the column is:

```css
@include media-breakpoint-down(md) {
    .js_product .td-img
```

Reply:
Oh wow I am such a moron, I hadn't even noticed.